src/runner_test.go

```diff
@@ -87,7 +87,7 @@ func TestVerifyYamlFile(t *testing.T) {
 			verificationCommand:          "false",
 			verificationArgs:             []string{},
 			shouldDoInsightsCoreGPGCheck: false,
-			expectedResult:               false,
+			expectedResult:               true,
```

Comment:
Would rename the name of the test case to something like `"GPG is disabled, verification should be skipped = considered valid"`

Reply:
Reverted the test as we will still support that option for now.